app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt

```diff
@@ -81,7 +80,7 @@ class MigratingFenixApplication : FenixApplication() {
             .migrateSettings()
             .build()
 
-        runBlocking {
+        runBlockingCounter {
```

Comment:
This name is inconsistent with others: `runBlockingIncrement`. Something is wrong.